identity/selector/handler.go

```diff
@@ -50,7 +50,27 @@ func NewHandler(
 	}
 }
 
-func (h *handler) UseExisting(address, passphrase string) (id identity.Identity, err error) {
+func (h *handler) UseOrCreate(address, passphrase string) (id identity.Identity, err error) {
+	if len(address) > 0 {
+		return h.useExisting(address, passphrase)
+	}
+
+	identities := h.manager.GetIdentities()
+	if len(identities) == 0 {
+		id, err = h.useNew(passphrase)
```

Comment:
We need to do something with this error. 
```suggestion
		return h.useNew(passphrase)
```